doc/classes/DisplayServer.xml

```diff
@@ -1086,12 +1086,18 @@
 		<constant name="FEATURE_SWAP_BUFFERS" value="18" enum="Feature">
 		</constant>
 		<constant name="MOUSE_MODE_VISIBLE" value="0" enum="MouseMode">
+			Makes the mouse cursor visible if it is hidden.
 		</constant>
 		<constant name="MOUSE_MODE_HIDDEN" value="1" enum="MouseMode">
+			Makes the mouse cursor hidden if it is visible.
 		</constant>
-		<constant name="MOUSE_MODE_CAPTURED" value="2" enum="MouseMode">
+		<constant name="MOUSE_MODE_CAPTURED" value="3" enum="MouseMode">
+			Captures the mouse. The mouse will be hidden and its position locked at the center of the screen.
+			[b]Note:[/b] If you want to process the mouse's movement in this mode, you need to use [member InputEventMouseMotion.relative].
 		</constant>
-		<constant name="MOUSE_MODE_CONFINED" value="3" enum="MouseMode">
+		<constant name="MOUSE_MODE_CONFINED" value="4" enum="MouseMode">
+			Makes the mouse cursor visible but confines it to the game window.
+			This is a bitwise enum, MOUSE_MODE_CONFINED you can hide the cursor with [code]MOUSE_MODE_CONFINED | MOUSE_MODE_HIDDEN[/code]
```

Comment:
```suggestion
			This is a bitwise enum. To confine and hide the cursor, use [code]MOUSE_MODE_CONFINED | MOUSE_MODE_HIDDEN[/code] as the mouse mode.
```

Reply:
Yeah, this documentation doesn't make sense as-is, it might have been a rebase error on my part.